features/403.Securing_Application_Access.feature

```diff
@@ -0,0 +1,81 @@
+Feature: Test tutorial 403.Securing application access
+    This is feature file of the FIWARE step by step tutorial for Managing roles and permissions
+    url: https://fiware-tutorials.readthedocs.io/en/latest/securing-access.html
+    git-clone: https://github.com/FIWARE/tutorials.Securing-Access.git
+    git-directory: /tmp/tutorials.Securing-Access
+    shell-commands: ./services create; ./services start
+    clean-shell-commands: ./services stop
+
+    Background:
+        Given I set the tutorial 403
+
+    Scenario: 00 - Generate Authorization header value
+        Given I set "ClientID" to "tutorial-dckr-site-0000-xpresswebapp"
+        And   I set "ClientSecret" to "tutorial-dckr-site-0000-clientsecret"
+        When  I calculate the base64 of this ClientId and ClientSecret
+        Then  I obtain the value "dHV0b3JpYWwtZGNrci1zaXRlLTAwMDAteHByZXNzd2ViYXBwOnR1dG9yaWFsLWRja3Itc2l0ZS0wMDAwLWNsaWVudHNlY3JldAo="
```

Comment:
The base64 seems to be wrong. It should end with "==" (no new line at the end) instead of "o=" which implies a new line at the end. The scenario is failing for that reason.